simplified-app-ekirjasto/src/main/assets/txnative/fi/txstrings.json

```diff
@@ -1453,7 +1723,7 @@
          "string" : "Varaukset"
       },
       "tabMagazines" : {
-         "string" : "Lehdet"
+         "string" : "Aikakauslehdet"
```

Comment:
This text is too long for the tab, should be changed back to "Lehdet" in Transifex.